research-hub-web/src/app/components/casestudys/case-study.component.html

```diff
@@ -1,115 +1,173 @@
-                <ng-container *ngTemplateOutlet="breadcrumbs"></ng-container>
-
-                <div fxLayout="row" fxLayoutAlign="space-between" fxLayout.lt-md="column wrap" fxLayoutGap="5%">
-
-                    <!-- caseStudy Body and References -->
-                    <div fxFlexOrder="2" fxFlexOrder.xs="1" fxFlex>
-                        <ng-container *ngTemplateOutlet="caseStudyBody"></ng-container>
-                        <ng-container *ngTemplateOutlet="caseStudyReferences"></ng-container>
-                    </div>
-
-                    <!-- Extra Links -->
-                    <div *ngIf="
-                            caseStudy.relatedDocsCollection.items.length > 0 ||
-                            caseStudy.relatedContactsCollection.items.length > 0 ||
-                            caseStudy.relatedOrgsCollection.items.length > 0"
-                        fxLayout="column" fxFlex="20%" fxFlexOrder="1" fxFlexOrder.lt-md="2">
-
-                        <!-- Contacts -->
-                        <ng-container *ngTemplateOutlet="relatedContacts; context: { flex: 'column', hide: 'true' }"></ng-container>
-
-                        <!-- Official Documents -->
-                        <ng-container *ngTemplateOutlet="officialDocuments; context: { flex: 'column', hide: 'true'}"></ng-container>
-
-                        <!-- Organizations -->
-                        <ng-container *ngTemplateOutlet="relatedOrganisations; context: { flex: 'column', hide: 'true'}"></ng-container>
+  <div
+    class="banner-container"
+    [ngClass]="
+      bannerImageUrl === undefined
+        ? ''
+        : isMobile
+        ? 'mobile-banner'
+        : 'standard-banner'
+    "
+    [ngStyle]="
+      bannerImageUrl && { 'background-image': 'url(' + bannerImageUrl + ')' }
+    "
+  >
+    <div class="banner-content">
+      <!-- Title -->
+      <h1
+        role="heading"
+        aria-level="1"
+        class="content-title"
+        ngClass.lt-md="content-title-mobile"
+        [style]="caseStudy.banner ? bannerTextStyling : ''"
+      >
+        {{ caseStudy.title }}
+      </h1>
+
+      <!-- Maori Proverb -->
+      <h2
+        aria-level="2"
+        *ngIf="caseStudy.maoriProverb"
+        class="maori-proverb"
+        [style]="caseStudy.banner ? bannerTextStyling : ''"
+      >
+        <hr />
+        {{ caseStudy.maoriProverb }}
+        <hr />
+      </h2>
+
+      <!-- Summary -->
+      <p
+        class="content-summary"
+        ngClass.sm="content-summary-sm"
+        ngClass.xs="content-summary-xs"
+        [style]="caseStudy.banner ? bannerTextStyling : ''"
+      >
+        <small>{{ caseStudy.summary }}</small>
+      </p>
+    </div>
+  </div>
+
+  <div class="site-padding" ngClass.lt-md="site-padding-mobile">
+    <app-breadcrumbs
+      [contentItem]="parentSubHubs"
+      [title]="caseStudy.title"
+    ></app-breadcrumbs>
+
+    <div fxLayout="row" fxLayoutAlign="space-evenly" fxLayout.lt-lg="column">
```

Comment:
In the three column layout (same on articles) might just need a small amount of margin added to the right of the body text. It's almost touching the cards at ~1300px screen width